flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlTableColumn.java

```diff
@@ -109,6 +137,14 @@ public void setType(SqlDataTypeSpec type) {
 		this.type = type;
 	}
 
+	public SqlNode getExpr() {
```

Comment:
Better to return `Optional<SqlNode>` to keep consistent with other nullable methods.